src/components/ProgressButton/ProgressButton.tsx

```diff
@@ -69,11 +62,11 @@ export const ProgressButton = ({
   }, [delayMs, serviceContainer.loading]);
 
   return (
-    <TooltipWrapper tooltip={isErrorState ? serviceContainer.error : undefined} isVisible={isTooltipVisible}>
```

Comment:
the other issue is, you handle only serviceContainer error state for Progress Button inside SSH button, not for Build button, where no tooltip is displayed on build fail.

```
isDisabled={!!disabledReason}
disabledTooltip={disabledReason}
```
to these two props, I would pass reason for disabled state relevant only to that higher level button and handle service container error in the progress button directly (and merge the tooltips and isDisabled in progress button)